### PR TITLE
fix bug: redict v2 job to default submission papge if plugin not installed

### PR DESCRIPTION
**Pull request by Binyang2014**

### CODE REVIEW

File: src/webportal/src/app/job-submission/job-submission.jsx [hunk 1]
```diff
@@ -244,6 +244,9 @@ const JobSubmission = () => {
               {vcNames}
             );
             jobInfo.name = generateJobName(jobInfo.name);
+            if (get(jobConfig, 'extras.submitFrom')) {
+              delete jobConfig.extras.submitFrom;
```
Comment (abuccts) [inline src/webportal/src/app/job-submission/job-submission.jsx]: should not delete

Comment (Binyang2014) [reply src/webportal/src/app/job-submission/job-submission.jsx]: The config is cloned from a plugin. But if resubmit, the config should indicate it submitted from the default submission page. So I think remove is the correct behavior

File: src/webportal/src/app/job/job-view/fabric/job-detail/conn.js [hunk 2]
```diff
@@ -134,6 +134,11 @@ export async function cloneJob(rawJobConfig) {
   const plugins = window.PAI_PLUGINS;
   const pluginIndex = plugins.findIndex((x) => x.id === pluginId);
   if (pluginIndex === -1) {
+    // redirect v2 job to default submission page
+    if (isJobV2(rawJobConfig)) {
+      window.location.href = `/submit.html?${qs.stringify(query)}`;
```
Comment (sunqinzheng) [inline src/webportal/src/app/job/job-view/fabric/job-detail/conn.js]: I think we should still pop up a warning message to user.

Comment (Binyang2014) [reply src/webportal/src/app/job/job-view/fabric/job-detail/conn.js]: sure, will fix it

**Pull request merged**